### ISSUE TITLE
No module name cleanup for Boolean Conditionals

### ISSUE THREAD

**Issue by mehiel** [systemjs/systemjs#503]
@guybedford I think the if statement in [contitionals.js#80](https://github.com/systemjs/systemjs/blob/master/lib/conditionals.js#L80) should change to:

```javascript
if (!conditionValue) {
  name = '@empty';
} else {
  name = name.replace(conditionalRegEx, '');
}
```

Otherwise name is passed with the condition included in normalize. You do clean the condition in case of substitution though, in [line 73](https://github.com/systemjs/systemjs/blob/master/lib/conditionals.js#L73).

Check the console output of this [jsfiddle](http://jsfiddle.net/j2Leb1jL/) for a minimal repro.

**Comment by guybedford** [comment]
Thank you, fixed in https://github.com/systemjs/systemjs/commit/72a67df4d93d1aaf3a49044f4cf783a6fab3f5d9.

**Comment by guybedford** [comment]
Released in SystemJS 0.18.
